public/app/features/transformers/extractFields/extractFields.ts

```diff
@@ -77,6 +77,10 @@ function addExtractedFields(frame: DataFrame, options: ExtractFieldsOptions): Da
       }
     }
 
+    if (obj == null) {
```

Comment:
I would move this up, at least around line 64.

Reply:
Well, `obj` can be changed in line 76, so this needs to be done here.

Reply:
Yep, but `const newObj: { [k: string]: unknown } = {};`. Not a big deal in any case 👍

Reply:
Good spot, Matias! Changed in d2d5dd588ddf93fa0e3bb460a99c194d08acb309